lepton/qemu.go

```diff
@@ -440,6 +440,7 @@ func (q *qemu) addAccel() {
 func (q *qemu) setConfig(rconfig *RunConfig) {
 	// add virtio drive
 	q.addDrive("hd0", rconfig.Imagename, "none")
+	// q.addOption("-drive", fmt.Sprintf("if=none,id=hd0,format=raw,file=%s", rconfig.Imagename))
```

Comment:
Did you forget to remove this commented out line?

Reply:
yes, fixed now